csharp/Platform.Collections/Arrays/ArrayPool[T].cs

```diff
@@ -21,15 +19,43 @@ public class ArrayPool<T>
         private readonly int _maxArraysPerSize;
         private readonly Dictionary<long, Stack<T[]>> _pool = new Dictionary<long, Stack<T[]>>(ArrayPool.DefaultSizesAmount);
 
+        /// <summary>
+        /// <para>Initializes a new instance of the ArrayPool class.</para>
+        /// <para>Инициализирует новый экземпляр класса ArrayPool.</para>
+        /// </summary>
+        /// <param name="maxArraysPerSize"><para>The maximum size of the array in the pool.</para><para>Максимальный размер массива в пуле.</para></param>
```

Comment:
`maxArraysPerSize` means "Maximum number of arrays per size", argument's comment should be fixed.
To understand the meaning of argument you should either look at how it is used, or ask the question.
[The argument is assigned to a field](https://github.com/linksplatform/Collections/blob/master/csharp/Platform.Collections/Arrays/ArrayPool%5BT%5D.cs#L25), and then [the field is used as an upper limit for each stack with arrays](https://github.com/linksplatform/Collections/blob/master/csharp/Platform.Collections/Arrays/ArrayPool%5BT%5D.cs#L60-L64). This ensures that we have not more than `maxArraysPerSize` arrays of each size.